person.rb

```diff
@@ -2,13 +2,14 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = generate_id
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
     super()
   end
```

Comment:
[ ] same thing here  Kindly add a method for adding rentals, the person itself should be sent as a parameter to create the new rental achieving the many associations. you can [Check the lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for that .🤓